pvlib/shading.py

```diff
@@ -342,3 +342,144 @@ def projected_solar_zenith_angle(solar_zenith, solar_azimuth,
+        horizontal. In degrees.
+    shading_tracker_tilt : numeric, optional
+        Tilt of the tracker casting the shadow.
+
+    Returns
+    -------
+    shaded_fraction : numeric
+        The fraction of the collector width shaded by an adjacent row. A
+        value of 1 is completely shaded and zero is no shade.
+
+    Notes
+    -----
+    All length parameters must have the same units to produce a reasonable
+    result.
+
+    Parameters are defined as follow:
+
+    .. figure:: ../../_images/Anderson_Jensen_2024_Fig3.png
+        :alt: Diagram showing the two trackers and the parameters of the model.
+
+        Figure 3 of [1]_. See correspondence between this nomenclature and the
+        function parameters in the table below.
+
+    +------------------+----------------------------+---------------------+
+    | Symbol           | Parameter                  | Units               |
+    +==================+============================+=====================+
+    | :math:`\theta_1` | ``shading_tracker_tilt``   |                     |
+    +------------------+----------------------------+                     |
+    | :math:`\theta_2` | ``shaded_tracker_tilt``    | Degrees             |
+    +------------------+----------------------------+ :math:`^{\circ}`    |
+    | :math:`\beta_c`  | ``cross_axis_slope``       |                     |
+    +------------------+----------------------------+---------------------+
+    | :math:`p`        | ``row_pitch``              | Any consistent      |
+    +------------------+----------------------------+ length unit across  |
+    | :math:`l`        | ``collector_width``        | all these           |
+    +------------------+----------------------------+ parameters, e.g.    |
+    | :math:`z_0`      | ``surface_to_axis_offset`` | :math:`m`.          |
+    +------------------+----------------------------+---------------------+
+    | :math:`f_s`      | Return value               | Dimensionless       |
+    +------------------+----------------------------+---------------------+
+
+    See also
+    --------
+    pvlib.tracking.projected_solar_zenith_angle
+
+    References
+    ----------
+    .. [1] Kevin S. Anderson, Adam R. Jensen; Shaded fraction and backtracking
+        in single-axis trackers on rolling terrain. J. Renewable Sustainable
+        Energy 1 March 2024; 16 (2): 023504. :doi:`10.1063/5.0202220`
+    """
+    # For nomenclature you may refer to [1].
+
+    # tilt of the tracker casting the shadow defaults to shaded tracker tilt
+    if shading_tracker_tilt is None:
+        shading_tracker_tilt = shaded_tracker_tilt
+
+    # projected solar zenith:
+    # consider the angle the sun direct beam has on the vertical plane which
+    # contains the row's normal vector, with respect to a horizontal line
+    projected_solar_zenith = projected_solar_zenith_angle(
+        solar_zenith,
+        solar_azimuth,
+        0,  # no rotation from the horizontal plane
```

Comment:
Does this have to be zero?  Can't we accept `axis_tilt` as a parameter as well?

Reply:
It doesn't. I just mixed concepts. Fixing it ASAP.